CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
```

Comment:
Please make sure with @netacoral that the changelog belongs here and not in appliance repo. Because when I added changelog here I had to move it later to appliance. I want to save you the trouble.
Also, if changhlog here is updated then the PR form needs to indicate it

Reply:
Amos and I just checked with her again and  she said its fine in conjur